<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datenbank</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="button_listenanzeige">
      <property name="text">
       <string>Listenanzeige</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_einzelanzeige">
      <property name="text">
       <string>Einzelanzeige</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_neuer_eintrag">
      <property name="text">
       <string>Neuer Eintrag</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_suche">
      <property name="text">
       <string>Suche</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
